Emojigrams/Emojigrams/EmojigramTitleScreenViewController.swift

```diff
@@ -11,8 +11,9 @@ import UIKit
 class EmojigramTitleScreenViewController: UIViewController {
     let emojigramStore = EmojigramStore()
     
-    override func viewDidLoad() {
-        super.viewDidLoad()
```

Comment:
#? even when changing to viewWillAppear, is it good practice to get rid of view didLoad all together?

Reply:
Yep! If you're not doing anything in the function, there's no reason for the parent classes implementation to be overridden. 

`override func viewDidLoad() { super.viewDidLoad }` is equivalent to not having it overridden at all.